src/esp/sim/AbstractReplayRenderer.cpp

```diff
@@ -91,12 +91,21 @@ void AbstractReplayRenderer::setSensorTransformsFromKeyframe(
 }
 
 void AbstractReplayRenderer::render(
-    Cr::Containers::ArrayView<const Mn::MutableImageView2D> imageViews) {
-  CORRADE_ASSERT(imageViews.size() == doEnvironmentCount(),
-                 "ReplayRenderer::render(): expected" << doEnvironmentCount()
-                                                      << "image views but got"
-                                                      << imageViews.size(), );
-  return doRender(imageViews);
+    Cr::Containers::ArrayView<const Mn::MutableImageView2D> colorImageViews,
+    Cr::Containers::ArrayView<const Mn::MutableImageView2D> depthImageViews) {
+  if (colorImageViews != nullptr) {
```

Comment:
I'm curious about the `nullptr` check here and then the use of default `std::vector<Mn::MutableImageView2D>{}` above. How does this work? If I omit `color_images` from my python call, do I get a zero-length `colorImageViews` here that fails this `ESP_CHECK`?

Reply:
Comparing `ArrayView` to `nullptr` checks for an empty array. I will standardize those avoid ambiguity.

Reply:
>Comparing ArrayView to nullptr checks for an empty array.
ಠ_ಠ